src/WebJobs.Script.WebHost/Security/SimpleWebTokenHelper.cs

```diff
@@ -18,12 +18,17 @@ public static class SimpleWebTokenHelper
         /// The SWT is then returned as an encrypted string
         /// </summary>
         /// <param name="validUntil">Datetime for when the token should expire</param>
+        /// <param name="key">Optional key to encrypt the token with</param>
         /// <returns>a SWT signed by this app</returns>
-        public static string CreateToken(DateTime validUntil) => Encrypt($"exp={validUntil.Ticks}");
+        public static string CreateToken(DateTime validUntil, byte[] key = null) => Encrypt($"exp={validUntil.Ticks}", key);
 
         internal static string Encrypt(string value, byte[] key = null)
         {
-            key = key ?? GetWebSiteAuthEncryptionKey();
+            if (key == null)
+            {
+                // This has to use WebSiteAuthEncryptionKey only. ContainerEncryptionKey will not be available on the FrontEnd.
```

Comment:
nit: This comment doesn't seem to belong in this class/method, since this is a general helper class used for tokens as well as encrypting/decrypting context payloads, etc.

Reply:
Fixed